packages/std/src/errors/std_error.rs

```diff
@@ -89,6 +89,19 @@ pub enum StdError {
         #[cfg(feature = "backtraces")]
         backtrace: Backtrace,
     },
+    #[error("Cannot add {operand1} to {operand2}")]
+    Overflow {
+        operand1: String,
+        operand2: String,
+        #[cfg(feature = "backtraces")]
+        backtrace: Backtrace,
+    },
```

Comment:
We should generalize this and remove the term "underflow" because:

> Storing values that are too low in an integer variable (e.g., attempting to store −1 in an unsigned integer) is properly referred to as integer overflow, or more broadly, integer wraparound. The term underflow normally refers to floating point numbers only, which is a separate issue. It is not possible in most floating-point designs to store a too-low value, as usually they are signed and have a negative infinity value. 

https://en.wikipedia.org/wiki/Arithmetic_underflow

Reply:
done ;D